packages/react/src/use-chat.ts

```diff
@@ -8,17 +8,24 @@ import type {
   UseChatOptions,
```

Comment:
review `packages/react/src/use-chat.ui.test.tsx` to see if there are additional test cases to add